pkg/dao/repository_configs.go

```diff
@@ -383,19 +388,26 @@ func (r repositoryConfigDaoImpl) Fetch(orgID string, uuid string) (api.Repositor
 	return repo, nil
 }
 
-func (r repositoryConfigDaoImpl) fetchRepoConfig(orgID string, uuid string) (models.RepositoryConfiguration, error) {
+// fetchRepConfig: "any" allows the fetching of red_hat repositories
+func (r repositoryConfigDaoImpl) fetchRepoConfig(orgID string, uuid string, any bool) (models.RepositoryConfiguration, error) {
```

Comment:
Instead of calling this boolean "any", how about "includeRedHatRepos" or something like that? Since that's specifically what it's deciding.

Also, "any" in Go is a keyword and alias for the type interface{}

Reply:
Yeah, I probably should have expected that. will do.